### PR TITLE
Fixing complete step during setup would always be in EN language

### PR DESCRIPTION
**Pull request by sdrenth**
### What does it do?
It sets the language setting again after clear installation setting so the complete step is shown in correct language.

### Why is it needed?
During complete step in setup the settings are cleared, which would also clear the language setting. Thats why the complete step would always be shown in English.

### Related issue(s)/PR(s)
Issue #13611

### CODE REVIEW

**Comment by gpsietzema** [comment]
I'll go test this now at the bug hunt.

**Comment by gpsietzema** [comment]
@sdrenth I tested your fix, but still end up with the default en-thankyou page:

screen shot 2018-02-09 at 10 58 56

What I did:
- New install within your branch
- Clear caches (browser/core) 
- Run setup in incognito
- Choose 'nl' at the first-setup page
- End up at an English thankyou-page

**Comment by Mark-H** [comment]
Also confirmed this doesn't fix the problem in my testing.

**Comment by sdrenth** [comment]
@gpsietzema @Mark-H After some more testing it sometimes did and sometimes didn't work. 

I already found the cause of the problem and will update the PR. The problem was with an incorrect check which led to a language cookie being set incorrectly.

**Comment by sdrenth** [comment]
I've updated the PR. Thanks for testing guys!

**Comment by gpsietzema** [comment]
Will test :)

**Comment by gpsietzema** [comment]
I can confirm it works now. Awesome work! 👍

File: setup/controllers/language.php [hunk 1]
```diff
@@ -7,12 +7,15 @@
  */
 /* parse language selection */
 if (!empty($_POST['proceed'])) {
-    $language= 'en';
-    $cookiePath = preg_replace('#[/\\\\]$#', '', dirname(dirname($_SERVER['REQUEST_URI'])));
```
Comment (Mark-H) [inline setup/controllers/language.php]: The cookie path is still being used below on line 17 (15), which is now undefined.

**Comment by sdrenth** [comment]
@Mark-H Thanks! I've fixed that with the latest commit

**Pull request merged**